src/rviz_visual_tools.cpp

```diff
@@ -222,7 +222,7 @@ bool RvizVisualTools::loadRvizMarkers()
   // ID
   sphere_marker_.ns = "Sphere";
   // Set the marker type.
-  sphere_marker_.type = visualization_msgs::Marker::SPHERE_LIST;
+  sphere_marker_.type = visualization_msgs::Marker::SPHERE;
```

Comment:
this would break the ``publishSpheres`` function. we should add a new marker type (perhaps rename this one sphere_marker_list)